core/include/scipp/core/element/comparison.h

```diff
@@ -91,44 +91,44 @@ constexpr auto equal = overloaded{
 constexpr auto not_equal =
     overloaded{comparison, [](const auto &x, const auto &y) { return x != y; }};
 
-constexpr auto max_equals =
-    overloaded{arg_list<double, float, int64_t, int32_t, bool>,
-               transform_flags::expect_in_variance_if_out_variance,
-               [](auto &&a, const auto &b) {
-                 using std::max;
-                 a = max(a, b);
-               }};
+constexpr auto max_equals = overloaded{
+    arg_list<double, float, int64_t, int32_t, bool, core::time_point>,
```

Comment:
These changes look similar to @jl-wynen 's PR #1747
we should probably coordinate?

Reply:
Good point, I forgot...

Reply:
Switched to using @jl-wynen 's implementation of `creation.h`, which is cleaner than what I had.

Reply:
in his PR he seems to not need the `core::` namespace specification here..